templates/webhook.bin.erb

```diff
@@ -253,14 +253,9 @@ class Server < Sinatra::Base
         # the other one to complete.
         file.flock(File::LOCK_EX)
 
-        if Open3.respond_to?('capture3')
-          stdout, stderr, exit_status = Open3.capture3(command)
-          message = "triggered: #{command}\n#{stdout}\n#{stderr}"
-        else
-          message = "forked: #{command}"
-          Process.detach(fork{ exec "#{command} &"})
-          exit_status = 0
-        end
+        message = "forked: #{command} in the background"
```

Comment:
`message = "triggered: #{command}"` would be better here IMO.

Reply:
Updated